TechLadies/irrc-madwish
# Export tables to CSV

# Export tables to CSV
by @therealyingtong @dzldebby @xuanjane 


([IRRC-14](https://bootcamp-irrc.atlassian.net/browse/IRRC-14)) Export student and teacher information
([IRRC-123](https://bootcamp-irrc.atlassian.net/browse/IRRC-123)) [Matched table] Add checkboxes to enable multiple selection for export purpose
Branch is based on #112 


## Description

Exports table information in each page (Screening, Matched, Students, Teachers) to CSV format.


## Test Steps

1. `npm install xlsx` on the frontend to install the xlsx library
2. Go to each page (Screening, Matched, Students, Teachers)
3. Click the Download/Download All/Download Selected button
4. Check that the information is downloaded into a CSV file
